enhancements/ingress/admin-configured-ca-bundle.md

```diff
@@ -0,0 +1,314 @@
+
+## Motivation
+
+When an OpenShift administrator creates a route with the re-encrypt termination type, they have to also
+provide a CA certificate for that route if it uses a custom CA instead of OpenShift's service CA for
+certificate validation. In addition, when the same CA is used by the administrator for signing several
+services' certificates it is time-consuming and error-prone to configure the same CA certificate in every
+created route object. Instead, this enhancement proposes a simpler alternative: in which an administrator
+could define a CA bundle containing the list of trusted CA certificates as a ConfigMap which will be used
+by OpenShift Router for verifying the services' certificates. Likewise, we'd like to improve the process
+for updating a CA certificate in each route object when the certificate is expired, compromised, or for
+any other reason in need of replacement.
+
+### User Stories
+- As an OpenShift cluster administrator, I want to configure a custom CA certificate bundle which OpenShift
+  router should use for verifying services' certificates, instead of having to configure the same as
+  destination CA Certificate in each of the created route objects.
+- As an OpenShift developer, I want to create a CA trust bundle for OpenShift Router usage, which includes
+  the OpenShift service CA and the admin provided CA bundle as the default CA for verifying the service's
+  certificate when a destination CA certificate is not configured for a route.
+- As an OpenShift cluster administrator, I want OpenShift router to update its configuration whenever the
+  OpenShift service CA certificate or the admin-configured CA certificate bundle is updated or deleted.
+- As an OpenShift cluster administrator, I want OpenShift router to verify services' certificates with just
+  the configured destination CA Certificate and not the default destination CA certificate.
+
+### Goals
+
+- An OpenShift administrator can configure a CA certificate bundle and expect OpenShift router to use it
+  for verifying a service's certificate when destination CA Certificate isn't specified in the route object
+  created for the service.
+- Openshift router should seamlessly reload the OpenShift service CA certificate or the
+  administrator-configured CA certificate bundle when either is modified.
+
+### Non-Goals
+
+- OpenShift administrator defined CA certificate bundle will be used only for the traffic through OpenShift
+  router and for the routes created with re-encrypt termination type, any other scenario is outside the
+  scope of this proposal.
+- The common CA bundle created by the operator is for all ingress controllers usage, and defining a CA
+  bundle specific for each IngressController usage is out of scope.
+
+## Proposal
+
+Change OpenShift Ingress Operator to:
+1. Create a CA trust bundle which will contain the OpenShift service CA certificate and the custom
+   user-supplied CA certificates.
```

Comment:
further up the discussion is of cluster-admin.  Is this a different user or a cluster-admin.  if cluster-admin use the same noun please.

Reply:
> It could be any user who has access to create the `admin-ca-bundle` ConfigMap in the `openshift-config` namespace, not in particular one with cluster-admin role assigned. Please let me know if it should be changed to `cluster-admin supplied`.

Which user other than a cluster-admin could do this in a stock cluster?

Reply:
Users with `admin` or `edit` roles could also create ConfigMaps in a given project.